jsk_pcl_ros/test/pointcloud_database_server.test

```diff
@@ -0,0 +1,17 @@
+<launch>
+  <include file="$(find jsk_pcl_ros)/sample/sample_pointcloud_database_server.launch">
+    <env name="DISPLAY" value="" />
```

Comment:
sorry i mean `bing.test`
I'm not used to make test, so i just follow the previous test.
https://github.com/jsk-ros-pkg/jsk_recognition/blob/master/jsk_perception/test/bing.test#L4
what is it for?